### ISSUE TITLE
Downloading youtube link consistently failing

### ISSUE THREAD

**Issue by pyromaniackeca** [sahlberg/pop-fe#208]
[Link](https://www.youtube.com/watch?v=ffBfYxzFABo) in question (I tried a few other ones as well).
Also tested link works by pasting it from the terminal output into the browser.
Using the `-v` doesn't provide any more useful info.

I'm running using wsl and pretty sure it's not a network issue.

This isn't much to go off of unfortunately. 😞
Any tips on how I can investigate this further or what else I can provide would be greatly appreciated.

**Comment by pyromaniackeca** [comment]
Ok, I think I managed to figure it out by changing the code to print out the error.
That led me to [this](https://github.com/pytube/pytube/issues/1707#issuecomment-1672149709).

TL;DR:
Seems like the pytube repo is poorly maintained and you should probably switch to [pytubefix](https://github.com/JuanBindez/pytubefix).

Installing the tool and switching the import seems to fix it.

**Comment by sahlberg** [comment]
Good work.
Can you send me a git pull request to change pop-fe to use pytubefix?

so you get proper credit for it in the git history.
